test/asm/unique-id.err

```diff
@@ -22,10 +22,6 @@ while expanding symbol "warn_unique"
 warning: unique-id.asm(14) -> unique-id.asm::m(8): [-Wuser]
     _u4!
 while expanding symbol "warn_unique"
-error: unique-id.asm(15):
-    Macro argument '\@' not defined
-while expanding symbol "warn_unique"
 warning: unique-id.asm(15): [-Wuser]
-    !
+    _u7!
 while expanding symbol "warn_unique"
-error: Assembly aborted (1 error)!
```

Comment:
Wait, why's that?

Reply:
Oops, `fstk_Init` was calling `macro_UseNewUniqueID()` to say "generate a new ID when requested", when instead it should have said "don't generate any ID at this level".